README.md

```diff
@@ -163,20 +164,42 @@ const capRootXTC = await CapRoot.init({ tokenId, router: capRouter })
 const xtcTransactions = await capRootXTC.get_transactions()
 ```
 
-Or (if you doesn't have an instance of the router)
+Alternatively, for the case where you don't have a `CapRouter` instance:
 
 ```js
 const tokenId = 'aanaa-xaaaa-aaaah-aaeiq-cai'	// XTC Canister Id
 
 const capRootXTC = await CapRoot.init({
 	tokenId,
 	routerCanisterId: 'rrkah-fqaaa-aaaaa-aaaaq-cai',
-	hostRouter: 'http://localhost:8000'
+	host: 'http://localhost:8000'
```

Comment:
@aleacevedo see comment bellow in the interface, as this uses common `host` (CapRouter and CapRoot)